src/NuGet.Core/NuGet.Packaging/Strings.resx

```diff
@@ -859,4 +859,10 @@ Valid from:</comment>
     <value>Some framework assembly reference TFMs are missing a platform version: {0}</value>
     <comment>0 - comma-separated list of reference group TFMs</comment>
   </data>
+  <data name="ZipFileTimeStampModified" xml:space="preserve">
+    <value>Last write timestamp for '{0}' changed from '{1}' to '{2}' because the zip file format does not support timestamp values before 01/01/1980 or after 12/31/2107.</value>
```

Comment:
outside of the US 12/31/2107 means the 12th day of the 31st month, which is invalid. If we're going to put this in the message, we should use more `{x}` substitutions, so that the culture-aware date formatter can format the date in the correct format.

Reply:
Ok. Taken your recommendation. Please check.